src/components/templates/Button/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+
+const Button = ({
+  backgroundColor = '#F74F2A',
+  color = '#FFFFFF',
+  borderRadius = '3px',
+  width = '324px',
+  height = '50px',
+  fontSize = '16px',
+  cursor = 'pointer',
+  className,
+  children,
+  onClick,
+  ...props
+}) => {
+  const buttonStyle = {
+    backgroundColor,
+    color,
+    borderRadius,
+    width,
+    height,
+    fontSize,
+  }
+  const Click = e => {
+    e.prevenDefault()
+  }
+  return (
```

Comment:
```suggestion
  }
  
  const handleClick = e => {
    e.prevenDefault()
    onClick()
  }
  
  return (
```

여백 정리, 변수명 수정, `onClick()`함수 실행 추가되면 좋을 것 같습니다!